config/complete/orderly-web.yml

```diff
@@ -73,7 +81,8 @@ orderly:
   image:
     repo: vimc
     name: orderly.server
-    tag: master
+    tag: vimc-4192
```

Comment:
Do we want to hold off on merging this until vimc-4192 is merged on orderly.server? I guess you need this to fix the client

Reply:
Yep - I think the order will be (updating references at each stage) 
* orderly.server vimc-4192 vimc/orderly.server#35
* orderlyweb mrc-2165 vimc/orderly-web#295
* This PR
* A new orderlyweb R package PR - TODO
* Possibly also a change to python client